sources/imperative/reader/Reader.swift

```diff
@@ -52,13 +52,22 @@ public final class CSVReader: IteratorProtocol, Sequence {
       guard !headers.isEmpty else { throw Error._invalidEmptyHeader() }
       self.headers = headers
       self.count = (rows: 1, fields: headers.count)
+    case let .lineNumber(index):
+      // Parse rows, but ignore them
+      for _ in 0..<index {
+        _ = try self._parseLine(rowIndex: 0)
+      }
+      guard let headers = try self._parseLine(rowIndex: 0) else { self.status = .finished; return }
+      guard !headers.isEmpty else { throw Error._invalidEmptyHeader() }
+      self.headers = headers
+      self.count = (rows: 0, fields: headers.count)
```

Comment:
`rows` should be `1` here. Similar to the `.firstLine` case. Therefore:
```swift
self.count = (rows: 1, fields: headers.count)
```

Reply:
Alright, fixed in 7e92ed9516d25b231964e4984cff7287202e64b3